src/shogun2-web/src/main/java/de/terrestris/shogun2/rest/UserRestController.java

```diff
@@ -46,4 +54,23 @@ protected UserRestController(Class<E> entityClass) {
 	public void setService(S service) {
 		this.service = service;
 	}
+
+	/**
+	 * Get the groups of a specific user.
+	 *
+	 * @param userId
+	 * @return
+	 */
+	@RequestMapping(value = "/{userId}/userGroups", method = RequestMethod.GET)
+	public @ResponseBody Map<String, Object> findGroupsOfUser(@PathVariable Integer userId) {
```

Comment:
As this is a REST interface we should return the entities, but not Map<String, Objects, which is based on ResultSet.success(...), which is an Ext-JS specific format, isn't it?